sdk/cosmos/azure-cosmos-spark_3-0_2-12/scalastyle-output.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this is spark code. remove please.

Reply:
oops, wrong checkin. Will remove in the next iteration.

Reply:
removed